src/modules/dashboard/components/vehicle-list.vue

```diff
@@ -274,11 +274,11 @@ export default Vue.extend<
             let missionName =
                 document.querySelector<HTMLAnchorElement>(
                     `#mission_caption_${id}`
-                )?.innerText ?? '';
```

Comment:
In which case is textContent better than innerText?

Reply:
there are multiple advantages such as better performance and more predictable behaviour.
It is enforced by ESLint-Rule [unicorn/prefer-dom-node-text-content](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-dom-node-text-content.md) to prefer `.textContent` over `.innerText`